policyengine-client/src/App.jsx

```diff
@@ -15,51 +20,70 @@ import LandingPage from "./landing";
 
 // Markdown files
 
-import UK_FAQ from "./markdown/uk/faq.md";
-import ABOUT from "./markdown/about.md";
-import CONTACT from "./markdown/contact.md";
+import { html as UK_FAQ } from "./markdown/uk/faq.md";
```

Comment:
Interesting, does this improve performance?

Reply:
This particular bit.. is more an idiosyncrasy with this particular plugin: [vite-plugin-markdown](https://github.com/hmsk/vite-plugin-markdown). It has named exports instead of just exporting the whole thing as a string